app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
```

Comment:
- [ OPTIONAL] Kindly add a foreign key here as well to the comments and likes to relate it as shown in the diagram so It is highly recommended to add them I am assigning as optional because you added the relation but just you need to refer it to the class 💯🔥